api/HTMLInputElement.json

```diff
@@ -318,10 +318,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": true
+              "version_added": false
```

Comment:
Safari does in fact have `HTMLInputElement.prototype.formEnctype`. I guess the problem here is that we have a typo in our data: we have `formEncType` — capital _T_ — but the actual member name is `formEnctype` — lowercase _t_.

Reply:
Good catch! We were generating tests for both:
http://mdn-bcd-collector.appspot.com/tests/api/HTMLInputElement/formEnctype
http://mdn-bcd-collector.appspot.com/tests/api/HTMLInputElement/formEncType

I've sent https://github.com/foolip/mdn-bcd-collector/pull/807 to remove the custom IDL for the miscapitalized version.

@vinyldarkscratch can you (separately) rename this entry?